packages/compass-query-bar/src/components/query-option.tsx

```diff
@@ -33,7 +33,7 @@ const queryOptionLabelStyles = css({
 });
 
 const documentEditorQueryOptionLabelStyles = css(queryOptionLabelStyles, {
-  minWidth: spacing[5] * 3,
+  minWidth: spacing[5] * 2,
```

Comment:
I reduced this in line with this thread https://mongodb.slack.com/archives/GDTJXPHD0/p1709576196037729

We don't have to align with the Filter label anymore because it is not there. There's often also no queries dropdown.